community-content/pytorch_pre_built_images_deployment/pytorch-text-classification-vertex-ai-deploy.ipynb

```diff
@@ -0,0 +1,1658 @@
+{
```

Comment:
<p>"[your-multi-region]-docker.pkg.dev/vertex-ai/prediction/pytorch-cpu.1-11:latest"&nbsp; -&gt;</p><p> </p><p>f"{REGION}-docker.pkg.dev/vertex-ai/prediction/pytorch-cpu.1-11:latest"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

Reply:
We are not able to use `REGION` (e.g. `us-central1`) since the images are in multi-regions (e.g. `us`).

Reply:
f'{REGION.split('-')[0]} ... `

Although we'd have to verify that we have images in said places. Otherwise, asis is fine.

Reply:
The mapping is more difficult since we have australia-* which maps to asia and northamerica-* and southamerica-* which map to us. Will leave it asis.